docs/developers/documentation/index.md

```diff
@@ -0,0 +1,209 @@
+
+If you're adding a document
+to an existing group, simply add a new `- file:` entry in the appropriate spot. For example, if I wanted to add 
+a `progress_bars.md` how to guide, I would place it in `napari/docs/howtos` and update `_toc.yml` as below:
+
+```yml
+- file: howtos/index
+subtrees:
+- titlesonly: True
+entries:
+- file: howtos/layers/index
+subtrees:
+- titlesonly: True
+    entries:
+    - file: howtos/layers/image
+    - file: howtos/layers/labels
+    - file: howtos/layers/points
+    - file: howtos/layers/shapes
+    - file: howtos/layers/surface
+    - file: howtos/layers/tracks
+    - file: howtos/layers/vectors
+- file: howtos/connecting_events
+- file: howtos/napari_imageJ
+- file: howtos/docker
+- file: howtos/perfmon
+- file: howtos/progress_bars # added
+```
+
+To create a new subheading, you need a `subtrees` entry. For example, if I wanted to add `geo_tutorial1.md` and `geo_tutorial2.md`
+to a new `geosciences` subheading in tutorials, I would place my documents in a new folder `napari/docs/tutorials/geosciences`,
+together with an `index.md` that describes what these tutorials would be about, and then update `_toc.yml` as below:
+
+```yml
+- file: tutorials/index
+subtrees:
+- entries:
+    - file: tutorials/annotation/index
+    subtrees:
+    - entries: 
+        - file: tutorials/annotation/annotate_points
+    - file: tutorials/processing/index
+    subtrees:
+    - entries:
+        - file: tutorials/processing/dask
+    - file: tutorials/segmentation/index
+    subtrees:
+    - entries:
+        - file: tutorials/segmentation/annotate_segmentation
+    - file: tutorials/tracking/index
+    subtrees:
+    - entries:
+        - file: tutorials/tracking/cell_tracking
+    - file: tutorials/geosciences/index                 # added
+    subtrees:                                           # added
+    - entries:                                          # added
+        - file: tutorials/geosciences/geo_tutorial1     # added
+        - file: tutorials/geosciences/geo_tutorial2     # added
+```
+
+## 5. Preview your document
+
+Once you've added your document to the `docs` folder and updated the `_toc.yml`, you can preview the website
+ locally by installing napari and the required dependencies, then running `make docs` from the root of
+the `napari` repository.
+
+```{warning}
+It's best to run these commands in a fresh conda environment!
+```
+
+```bash
+pip install ".[all, dev, docs]"
+make docs
+```
+
+The rendered HTML will be placed in `napari/docs/_build`. Find `index.html` in this folder and drag it
+into a browser to preview the website with your new document.
+## 6. Submit your pull request
+
+Once you have written and previewed your document, it's time to open a pull request to [napari's main repository](https://github.com/napari/napari) and contribute it to our codebase. 
+
+Not sure where to place your document or update `_toc.yml`? Make a best guess and open the pull request - the napari team will
```

Comment:
I think we need to link to "how to make a pull request", e.g. this one is good (a bit better than github's I think):

https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github

But here is github's for reference:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request